tensorflow_addons/rnn/cell.py

```diff
@@ -16,6 +16,11 @@
 
 import tensorflow as tf
 import tensorflow.keras as keras
+from typeguard import typechecked
+
+from tensorflow_addons.utils import types
+from typing import Optional
+from tensorflow_addons.utils.types import Activation
```

Comment:
You could use `types.Activation` instead of this import.

Reply:
Yeah, thanks @guillaumekln.